src/devices/nodon.ts

```diff
@@ -54,6 +108,60 @@ const definitions: Definition[] = [
             await reporting.currentPositionLiftPercentage(endpoint);
             await reporting.currentPositionTiltPercentage(endpoint);
         },
+        extend: [
+            enumLookup({
```

Comment:
Since this is duplicated, can we define a `const nodonModernExtend` in the top of the file and move them to there?